go1.13_compat_test.go

```diff
@@ -0,0 +1,18 @@
+// +build go1.13
+
+package errors_test
+
+import (
+	stdlib_errors "errors"
+	"testing"
+
+	"github.com/pkg/errors"
+)
+
+func TestErrorChainCompat(t *testing.T) {
+	err := stdlib_errors.New("error that gets wrapped")
+	wrapped := errors.Wrap(err, "wrapped up")
```

Comment:
```suggestion
wrapped := Wrap(err, "wrapped up")
```

Reply:
Please show some care when proposing suggestions that the suggestion if plainly applied will not break go formatting.